README.md

```diff
@@ -30,10 +30,20 @@ Install npm dependencies:
 
 ```
 npm install
-``
+```
+
+## Creating iteration reports
+
+First, edit the iteration definition file `iterations.json` to add the dates for the next iteration. Push the change to github to trigger the iteration summary script run.
 
+Then create a new iteration report using the post-bundle archetype with the filename for the post you want to create. By convention, use isodates for iteration reports. To autogenerate with the current date, use this command:
```

Comment:
Where does the term "post-bundle" come from? I see "page bundle" in the Hugo docs but not sure what this is a reference to - same thing? Would this archetype not just be called "iteration report"?

Reply:
You're right, it is a page bundle. I wonder where I got post bundle? I guess it's probably clearer to just call it an iteration report archetype? Will revise wording before I merge.

Reply:
It seems like the convention is to call it by the type of post/page it's a template for, so "iteration report archetype" makes sense to me!